extensions/amp-story/1.0/progress-bar.js

```diff
@@ -35,14 +37,47 @@ const TRANSITION_LINEAR = `transform ${POLL_INTERVAL_MS}ms linear`;
  */
 const TRANSITION_EASE = 'transform 200ms ease';
 
+/**
+ * Size in pixels of a segment ellipse.
+ * @const {number}
+ */
+const ELLIPSE_WIDTH_PX = 2;
```

Comment:
Yep sorry they should be `2px` on mobile and `3px` on both desktop versions, to reflect our CSS